doc/guide/cli_quickstart.hpp

```diff
@@ -10,7 +10,7 @@ This page describes how you can quickly get started using mlpack from the
 command-line and gives a few examples of usage, and pointers to deeper
 documentation.
 
-This quickstart guide is also available for @ref python_quickstart "Python".
+This quickstart guide is also available for @ref python_quickstart "Python" and @ref julia_quickstart "Julia".
```

Comment:
Pedantic, but let's try to keep it 80 lines.
```suggestion
This quickstart guide is also available for @ref python_quickstart "Python" and
@ref julia_quickstart "Julia".
```

Reply:
Thanks.